deploy-board/deploy_board/templates/accounts/accounts.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'accounts/accounts_base.html' %}
+
+{% block breadcrumb-items %}
+<ul class="breadcrumb">
+    <li><a href="/">Home</a></li>
+    <li class="active">Accounts</li>
+</ul>
+{% endblock %}
+
+{% block main %}
+<div class="panel panel-default">
+    <div class="panel-heading clearfix">
+        <h4 class="panel-title pull-left">Account Details</h4>
+    </div>
+    <div class="panel-body table-responsive">
+        <table class="table table-striped table-bordered table-condensed table-hover">
+            <thead>
+                <tr>
+                    <th>ID</th>
+                    <th class="col-lg-2">Name</th>
+                    <th class="col-lg-2">Cell</th>
+                    <th class="col-lg-2">Provider</th>
+                    <th class="col-lg-2"></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for account in accounts %}
+                <tr>
+                    <td>{{ account.id }}</td>
+                    <td>{{ account.name }}</td>
+                    <td>{{ account.cell }}</td>
+                    <td>{{ account.cloudProvider }}</td>
+                    <td><a href="/accounts/{{ account.cloudProvider }}/{{ account.cell }}/{{ account.id }}">Details</a></td>
+                </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+    </div>
+</div>
+{% endblock %}
```

Comment:
Please add empty line at the end of every new file. It can be achieved throughout IDE config, for example Intelij IDEA:
![Screenshot 2024-03-12 at 5 27 44 PM](https://github.com/pinterest/teletraan/assets/20383875/d43cdf48-07c4-4951-94e8-0c5a53f90877)